gradle/libs.versions.toml

```diff
@@ -72,6 +73,7 @@ lombok = { group = "org.projectlombok", name = "lombok", version.ref = "lombok"
 junit-jupiter-api = { group = "org.junit.jupiter", name = "junit-jupiter-api", version.ref = "junit" }
 junit-jupiter-params = { group = "org.junit.jupiter", name = "junit-jupiter-params", version.ref = "junit" }
 junit-jupiter-engine = { group = "org.junit.jupiter", name = "junit-jupiter-engine"}
+junit4 = { group = "junit", name = "junit", version.ref = "junit4"}
```

Comment:
Could we remove the junit4 dependency?

Reply:
Nope, because removing it will cause the HDFS MiniCluster initializes failed, it relies on Junit4.

Reply:
Maybe we can remove it after we upgrade the Hadoop dependency.